.coveragerc

```diff
@@ -0,0 +1,5 @@
+[run]
+branch = 1
+# NOTE: cannot use package easily, without chdir (https://github.com/nedbat/coveragepy/issues/268).
+source = paste/,tests/
```

Comment:
We don't want coverage reports for the tests do we? Just the code in paste/. Otherwise we're creating noise.

Unless you're trying to use coverage to find test code that isn't being reached, but I'm not sure if we want to focus on that.

Reply:
> Unless you're trying to use coverage to find test code that isn't being reached, but I'm not sure if we want to focus on that.

That's the main reason, yes - especially when testing compat code etc.. in other suite this even showed that some test code was accidentally never run etc.

> Otherwise we're creating noise.

Maybe this can be improved through config then, i.e. if you do not want to have it with "coverage report" by default, but this would then also affect "coverage xml" by default IIRC.

Reply:
I'm happy to leave it as is, and use it to improve the test code.